dashboard/config/blocks/storylab/storylab_setForegroundEffect2.js

```diff
@@ -1,381 +1,3 @@
-//This is the unused sprite we attach behaviors to, which is how we get animations to happen over time
-//When the animation is over, we remove the behavior from the sprite
-//createNewSprite({name: 'effectSprite'}, "", ({"x":-200,"y":-200}));
-
-//From https://github.com/code-dot-org/code-dot-org/blob/525025d9870b45a81dcb4c69f7541f4f2b1abeee/apps/src/p5lab/poetry/constants.js
-var PALETTES = {
-  grayscale: ['#000000','#333333','#666666','#999999','#CCCCCC','#EEEEEE','#FFFFFF'],
-  sky: ['#3878A4', '#82A9B1', '#ECCEC4', '#F8B8A8', '#E4929C', '#7D7095'],
-  ocean: ['#7FD0F5', '#3FABE3', '#2C7DBB', '#1D57A0', '#144188', '#061F4B'],
-  sunrise: ['#F5DC72', '#F4B94F', '#F48363', '#F15C4C', '#372031'],
-  sunset: ['#530075', '#921499', '#E559BB', '#F7B9DD', '#307087', '#123F50'],
-  spring: ['#303F06', '#385202', '#547607', '#85AF4C', '#C1E876', '#D7FF6B'],
-  summer: ['#FAD0AE', '#F69F88', '#EE6E51', '#BC4946', '#425D19', '#202E14'],
-  autumn: ['#484F0C', '#AEA82E', '#EEBB10', '#D46324', '#731B31', '#4A173C'],
-  winter: ['#EAECE8', '#E3DDDF', '#D3CEDC', '#A2B6BF', '#626C7D', '#A4C0D0'],
-  twinkling: ['#FFC702', '#FC9103', '#F17302', '#B83604', '#7E1301'],
-  rainbow: ['#A800FF', '#0079FF', '#00F11D', '#FF7F00', '#FF0900'],
-  roses: ['#4C0606', '#86003C', '#E41F7B', '#FF8BA0 ', '#FFB6B3']
-};
-
-function randomColorFromPalette(palette) {
-  return color(PALETTES[palette][randomInt(0, PALETTES[palette].length - 1)]);
-}
-
-var drops = [];
-var numDrops = 20;
-
-var numBubbles = 25;
-var bubbles = [];
-
-var numHearts = 15;
-var hearts = [];
-
-var numConfetti = 25;
-var confetti = [];
-
-var numStars = 30;
-var stars = [];
-
-var numTwinkleStars = 30;
-var twinkleStars = [];
-
-function randomInt(min, max) {
- 	return Math.floor(Math.random() * (max-min)) + min;
-}
-
-function randomColor() {
-  return color('hsba(' + randomNumber(0, 359) + ', 100%, 100%, 0.3)').toString();
-}
-
-/** Creates a rain effect with 20 drops on the screen
-*/
-function rainEffect() {
-  if (drops.length === 0) {
-    //removeBehaviorSimple(({name: "effectSprite"}), new Behavior(rainEffect, []));
-    return;
-  }
-  console.log("inside rain effect");
-  push();
-  stroke(
-  rgb(92, 101, randomInt(140, 220), 0.5));
-  strokeWeight(3);
-  for (var i = 0; i < drops.length; i++) {
-    push();
-    translate(drops[i].x - 20, drops[i].y - 20);
-    line(0, 0, drops[i].length, drops[i].length * 2);
-    drops[i].y = drops[i].y + drops[i].length;
-    drops[i].x = drops[i].x + drops[i].length / 2;
-    pop();
-  }
-  drops = drops.filter(function(drop) {
-    return (drop.y < 420 && drop.x < 420);
-  });
-  pop();
-}
-
-//From: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/p5lab/poetry/commands/foregroundEffects.js
 function setForegroundEffect2(option) {
```

Comment:
This is where we'd leave a comment for an engineer coming after you that we cannot delete this, right? Since we can't leave JSON comments on `dashboard/config/blocks/GamelabJr/gamelab_setForegroundEffect.json`. Do you mind adding one?